package.json

```diff
@@ -4,7 +4,8 @@
   "description": "TRIM API wrapper",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "./node_modules/mocha/bin/mocha specs.js",
+    "test:watch": "./node_modules/nodemon/bin/nodemon.js --exec \"npm run test\" specs.js"
```

Comment:
Have you seen testem? We've used it quite a bit to rerun tests when files change.